www-client/elinks/files/elinks-0.13.3-ruby-gcc10.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/scripting/ruby/core.h
```

Comment:
Is this upstreamed? Please add a link at the top of the file for where it came from.